geomesa-core/src/test/scala/org/locationtech/geomesa/core/data/AccumuloDataStoreTest.scala

```diff
@@ -1155,6 +1153,61 @@ class AccumuloDataStoreTest extends Specification {
       success
     }
 
+    "create key plan that uses STII Filter with bbox" in {
+      val sftName = "explainLargeBBOXTest_1"
+      val sft1 = createSchema(sftName)
+      val filter = CQL.toFilter("bbox(geom, -100, -45, 100, 45)")
+      val query = new Query(sftName, filter, Array("geom"))
+      val explain = {
+        val o = new ExplainString
+        ds.explainQuery(sftName, query, o)
+        o.toString()
+      }
+      explain must contain("STII Filter: [ geom bbox POLYGON ((-100 -45, -100 45, 100 45, 100 -45, -100 -45)) ]")
+    }
+
+    "create key plan that does not use STII when given the Whole World bbox" in {
+      val sftName = "explainLargeBBOXTest_2"
+      val sft2 = createSchema(sftName)
+      val filter = CQL.toFilter("bbox(geom, -180, -90, 180, 90)")
+      val query = new Query(sftName, filter, Array("geom"))
+      val explain = {
+        val o = new ExplainString
+        ds.explainQuery(sftName, query, o)
+        o.toString()
+      }
+      explain must contain("No STII Filter")
+      explain must contain("Filter: AcceptEverythingFilter")
+    }
+
+    "create key plan that does not use STII when given something larger than the Whole World bbox" in {
+      val sftName = "explainLargeBBOXTest_3"
+      val sft3 = createSchema(sftName)
+      val filter = CQL.toFilter("bbox(geom, -190, -100, 190, 100)")
+      val query = new Query(sftName, filter, Array("geom"))
+      val explain = {
+        val o = new ExplainString
+        ds.explainQuery(sftName, query, o)
+        o.toString()
+      }
+      explain must contain("No STII Filter")
+      explain must contain("Filter: AcceptEverythingFilter")
+    }
+
+    "create key plan that does not use STII when given an or'd geometry query both of the whole world" in {
+      val sftName = "explainLargeBBOXTest_4"
+      val sft3 = createSchema(sftName)
+      val filter = CQL.toFilter("bbox(geom, -180, -90, 180, 90) OR bbox(geom, -180, -90, 180, 90)")
```

Comment:
I'm not certain this test is the one we want.  Wouldn't it be better if each of the two geometries were smaller than the entire world, but their union covered the (lat, lon) space?